ms_db/appsettings.Development.json

```diff
@@ -0,0 +1,8 @@
+{
+  "Logging": {
+    "LogLevel": {
+      "Default": "Information",
```

Comment:
I can see you've added this 
but most of the code is written with console.writelines

consider moving them into ILogger instance